CONTRIBUTION.md

```diff
@@ -0,0 +1,23 @@
+Contributing
+============
+
+We have a contributors Slack room, kindly fill this contributors form to get added: https://docs.google.com/forms/d/1JiRNzYS69ojHPCql254HMMmp72cRb2TMGVC6BiOTw6E/viewform?c=0&w=1
+
+**The #1 contribution you could make is to blog, share, post, tweet, and tell people about disease-info.  This will go a long ways towards helping build a sustainable community.**
+
+We are happy to accept contributions of any kind, including feedback and ideas, translations for other locales, and functionality. For new functionality, follow the standard approach by filling the form above.
+
+Git Workflow
+============
+The array of possible workflows can make it hard to know where to begin when implementing Git in the workplace. This page provides a starting point by surveying the most common Git workflows for enterprise teams. As you read through, remember that these workflows are designed to be guidelines rather than concrete rules.
+
+5. Before you continue with this guideline, It is assumed that you have a basic knowledge of using version control. If not, you might want to take tutorials on that.
```

Comment:
We could add these links for the tutorials.

For a short but apt tutorial, see [here](https://learn.wheelhouse.io/events/workflow) - and for a more in-depth tutorial see [here](https://learn.wheelhouse.io/events/early-access)